core/store/orm/orm_test.go

```diff
@@ -1330,6 +1330,31 @@ func TestORM_SyncDbKeyStoreToDisk(t *testing.T) {
 	assert.Equal(t, key.JSON.String(), content)
 }
 
+func TestORM_RemoveUnstartedTransaction(t *testing.T) {
+	storeInstance, cleanup := cltest.NewStore(t)
+	defer cleanup()
+	ormInstance := storeInstance.ORM
+
+	runJob := cltest.NewJobWithRunLogInitiator()
+	require.NoError(t, storeInstance.CreateJob(&runJob))
+
+	for _, status := range []models.RunStatus{
+		models.RunStatusInProgress,
+		models.RunStatusUnstarted,
+	} {
+		jobRun := cltest.NewJobRun(runJob)
+		jobRun.Status = status
+		require.NoError(t, storeInstance.CreateJobRun(&jobRun))
+	}
+
+	assert.NoError(t, ormInstance.RemoveUnstartedTransactions())
+
+	jobRuns, err := ormInstance.JobRunsFor(runJob.ID, 10)
+	assert.NoError(t, err)
+	assert.Len(t, jobRuns, 1, "expected only one jobrun to be left in the db")
+	assert.Equal(t, jobRuns[0].Status, models.RunStatusInProgress)
+}
```

Comment:
can we also test for deleted task runs, run requests, eth txs?

Reply:
I thought about this, it's a good point but I couldn't find factories for all these models. I'll look in other tests.

Reply:
Create one if it doesn't exist

Reply:
Turns out I didn't have to, they were all in `core/internal/cltest/factories.go`